core/services/promreporter/prom_reporter.go

```diff
@@ -218,30 +215,10 @@ AND evm.txes.state = 'unconfirmed'`, head.EVMChainID.String()).Scan(&earliestUnc
 }
 
 func (pr *promReporter) reportPipelineRunStats(ctx context.Context) (err error) {
-	rows, err := pr.db.QueryContext(ctx, `
-SELECT pipeline_run_id FROM pipeline_task_runs WHERE finished_at IS NULL
-`)
+	pipelineTaskRunsQueued, pipelineRunsQueued, err := pr.txStore.GetPipelineRunStats(ctx)
```

Comment:
I would suggest not to touch this function. The TXM doesn't own these tables, and thus this shouldn't go inside the TXM.